Playgrounds/2b_Maybe_Type_Monad.playground/contents.swift

```diff
@@ -108,13 +108,13 @@ The use of *switch statements* would add verbosity and remove clarity.
 
 Notice that the *pure* function is used to lift the return value into the **Maybe** type.
 */
-func livingSpace(person: Maybe<Person>) -> Maybe<Int> {
+func livingSpace(_ person: Maybe<Person>) -> Maybe<Int> {
     return person >>= { $0.residence }
```

Comment:
hah, fixed it - as mark lacey said in SR-2318, just the `combine:` parameter label is too much, the swift compiler just does not give a good compile error diagnostic